portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
-*/
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function getMessages() {
+  fetch('/data').then(response => response.json()).then((messages) => {
+    const messagesElement = document.getElementById('messages-container');
+    console.log(messages);
+    for (var i = 0; i < messages.length; i++) {
```

Comment:
consider using a for/of loop, since the index isn't really important.

Reply:
Didn't know about this before, but I just learned about it [online](https://www.w3schools.com/js/js_loop_for.asp#:~:text=). Seems really cool!